translate-pixiv-tags.user.js

```diff
@@ -355,6 +458,24 @@ function safeMatch (string, regex, group = 0, defaultValue = "") {
 
 const safeMatchMemoized = _.memoize(safeMatch, memoizeKey);
 
+const TRANSLATE_PROFILE_URL = [{
```

Comment:
here we go again https://github.com/evazion/translate-pixiv-tags/commit/cf0b20621c29b6563c36afe06e677eeff26d4627

Reply:
I wasn't aware that Deviantart was still an issue. I thought that @nonamethanks converted all of the Deviantart links over to the new format some time back. However, I'll go ahead and add that one in. It'll probably also be good to add the Artstation one in as well.

Reply:
No, SauceNAO still provides DA link in the old style.